package.json

```diff
@@ -59,7 +60,6 @@
     "usehooks-ts": "^3.1.1"
   },
   "devDependencies": {
-    "@biomejs/biome": "1.9.4",
```

Comment:
Remove biome because although it is newer and said to be faster, I don't think it has Next.js support yet. Also for consistency with OutcomesTracker, I use eslint.

Reply:
Love it!